src/governance/governance.h

```diff
@@ -270,8 +270,8 @@ class CGovernanceManager : public GovernanceStore
     int nCachedBlockHeight;
     std::map<uint256, CGovernanceObject> mapPostponedObjects;
     hash_s_t setAdditionalRelayObjects;
-    hash_s_t setRequestedObjects;
-    hash_s_t setRequestedVotes;
+    std::map<uint256, int64_t> mapRequestedObjects;
```

Comment:
if `set` is giving information what that's just a set of requested object, the `map` as a member name prefix doesn't give a context.

Consider renaming mapRequestedObjects to `requestedObjectsWithTime` or `timeOfRequestedObjects` or something else.

Same for `AcceptMessage`: `mapHash` -> `hashesWithTime` ?

Reply:
Good point 👍  Also the logic for these 2 maps is essentially the same so I took it a bit further and squashed them into one, see dea1395.